app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +336,25 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_capacity(hash)
+      result = {}
+      hash.each do |key, val|
+        res = parse_capacity_field(key, val)
+        result[key] = res if res
+      end
+      result
```

Comment:
Minor, but this can use `each_with_object` to trim up the code...

```ruby
def parse_capacity(hash)
  hash.each_with_object({}) do |(key, val), result|
    res = parse_capacity_field(key, val)
    result[key] = res if res
  end
end
```